goshopify.go

```diff
@@ -169,6 +177,17 @@ func (c *Client) Do(req *http.Request, v interface{}) error {
 	return nil
 }
 
+func wrapSpecificError(r *http.Response, err ResponseError) error {
+	if err.Status == 429 {
+		f, _ := strconv.ParseFloat(r.Header.Get("retry-after"), 64)
```

Comment:
Yes I am sure. I forced a 429 on a test store and logged all the response headers (posted one on Slack for you :) )

And in fact, the message rate is *not* included in the 429 response (I'm assuming you are referring to the `X-Shopify-Shop-Api-Call-Limit: 29/40` header). It's only included during a successful request, so it can't be included in the error, unfortunately.